op-chain-ops/cmd/celo-migrate/main.go

```diff
@@ -59,25 +48,58 @@ var (
 		Usage:    "Path to write the rollup config JSON file, to be provided to op-node with the 'rollup.config' flag",
 		Required: true,
 	}
-	dbPathFlag = &cli.StringFlag{
-		Name:     "db-path",
-		Usage:    "Path to the Celo database, not including the `celo/chaindata` part",
+	oldDBPathFlag = &cli.PathFlag{
+		Name:     "old-db",
+		Usage:    "Path to the old Celo chaindata",
+		Required: true,
+	}
+	newDBPathFlag = &cli.PathFlag{
+		Name:     "new-db",
+		Usage:    "Path to write migrated Celo chaindata",
 		Required: true,
 	}
-	dryRunFlag = &cli.BoolFlag{
-		Name:  "dry-run",
-		Usage: "Dry run the upgrade by not committing the database",
+	batchSizeFlag = &cli.Uint64Flag{
+		Name:  "batch-size",
+		Usage: "batch size to use for block migration",
+		Value: 50000,
+	}
+	memoryLimitFlag = &cli.Int64Flag{
+		Name:  "memory-limit",
+		Usage: "Memory limit in MB",
+		Value: 7500,
+	}
+	clearAllFlag = &cli.BoolFlag{
+		Name:  "clear-all",
+		Usage: "Use this to start with a fresh new db, deleting all data including ancients. CAUTION: Re-migrating ancients takes time.",
+	}
+	keepNonAncientsFlag = &cli.BoolFlag{
+		Name:  "keep-non-ancients",
+		Usage: "CAUTION: Not recommended for production. Use to keep all data in the new db as is, including any partially migrated non-ancient blocks and state data. If non-ancient blocks are partially migrated, the script will attempt to resume the migration.",
```

Comment:
Given the description it seems like this flag would be better named `resume`, since the ancients are also being kept.

Reply:
Ok as discussed that doesn't make sense in the context of ancients since they are always resumed, so I think its ok to keep as a (non user flag) to help with testing.

Reply:
Discussed in call, we plan on deleting this flag (along with some others) that are not intended to be used for the actual migration but are useful for testing as we explore ways to shorten the mainnet migration downtime